ffi-convert/src/conversions.rs

```diff
@@ -138,15 +131,36 @@ pub struct UnexpectedNullPointerError;
 /// The `from_raw_pointer` effectively takes back ownership of the pointer. If you didn't create the
 /// pointer yourself, please use the `as_ref` method on the raw pointer to borrow it
 pub trait RawPointerConverter<T>: Sized {
+    /// Create a raw pointer from the value and leak it, you should use [`from_raw_pointer`] or
+    /// [`drop_raw_pointer`] to free the value when you're done with it.
     fn into_raw_pointer(self) -> *const T;
+    /// Create a mutable raw pointer from the value and leak it, you should use
+    /// [`from_raw_pointer_mut`] or [`drop_raw_pointer_mut`] to free the value when you're done
+    /// with it.
     fn into_raw_pointer_mut(self) -> *mut T;
+    /// Take back control of a raw pointer created by [`into_raw_pointer`].
+    /// # Safety
+    /// This method is unsafe because passing it a pointer that was not created by
+    /// [`into_raw_pointer`] can lead to memory problems. Also note that passing the same pointer
+    /// twice to this function will probably result in a double free
```

Comment:
```suggestion
    /// Takes back control of a raw pointer created by [`into_raw_pointer`].
    /// # Safety
    /// This method is unsafe because passing it a pointer that was not created by
    /// [`into_raw_pointer`] can lead to memory problems. Also note that passing the same pointer
    /// twice to this function will probably result in a double free
```